migrations/sql/V1.28.1__CE-1054.sql

```diff
@@ -0,0 +1,19 @@
+-- PURGE all the decision data that may have been stored to this point to avoid any issue with
+-- the uniqueness constraint to be added here.
+DELETE from case_management.action
+where
+  action.decision_guid is not null;
+
+DELETE from case_management.decision;
+
+DELETE from case_management.schedule_sector_xref;
+
+-- end PURGE of decision data
+-- Add uniqueness constraint to avoid repeatable seed data script from filling the DB with copies
+ALTER TABLE case_management.schedule_sector_xref ADD CONSTRAINT UK_schedule_sector_xref__schedule_sector UNIQUE (schedule_code, sector_code);
+
+UPDATE case_management.configuration
+SET
+  configuration_value = cast(configuration_value as INTEGER) + 1
+WHERE
+  configuration_code = 'CDTABLEVER';
```

Comment:
This isn't required for DDL changes.

You'll need to rename this script to 1.28.2 as this version number got snagged by another release.

Reply:
Renamed the file and removed the versioning statement. :-)